definitions/scenarios/upgrade_to_capsule_6_14.rb

```diff
@@ -58,7 +57,9 @@ def compose
       add_step(Procedures::Repositories::Setup.new(:version => '6.14'))
       modules_to_enable = ["satellite-capsule:#{el_short_name}"]
       add_step(Procedures::Packages::EnableModules.new(:module_names => modules_to_enable))
-      add_step(Procedures::Packages::UnlockVersions.new)
```

Comment:
Is unlocking no longer needed?

Reply:
Unlocking is designed to happen whenever an action requires it, explicitly unlocking here serves no purpose and further wastes time.